reference/forms/types/entity.rst

```diff
@@ -183,6 +183,32 @@ the entity and return an instance of ``QueryBuilder``.
 Overridden Options
 ------------------
 
+choice_name
+~~~~~~~~~~~
+
+.. versionadded:: 2.7
+    The ``choice_name`` option was introduced in Symfony 2.7.
+
+**type**: ``string``, ``callable`` or :class:`Symfony\\Component\\PropertyAccess\\PropertyPath` **default**: id
+
+By default the name of each field is the id of the entity, if it can be get in
```

Comment:
Reading this paragraph I wonder: what will be the default value if the "id reader" can't read the id of the object?